lib/kernel.ts

```diff
@@ -458,6 +458,12 @@ export class Kernel {
 			unsafeUpsert(CONTRACTS['relationship-org-has-member-user']),
 		]);
 
+		// Add all relationships
+		const relationshipContracts = _.values(CONTRACTS).filter(
+			(cd) => cd.type === 'relationship@1.0.0',
+		);
+		await Promise.all(relationshipContracts.map((cd) => unsafeUpsert(cd)));
+
```

Comment:
We're already upserting the relationships a couple lines above. I like the idea of using some logic to find and insert contracts based on type etc, but we should do that for all contracts and not make relationships a special case if we're going to go that route.

Reply:
Ok, making the change to insert relationships individually to be consistent with how other contracts are inserted at bootstrap.